backend/src/libs/packages/http/http-service.ts

```diff
@@ -0,0 +1,54 @@
+import { ContentType } from '#libs/enums/enums.js';
+import { type ValueOf } from '#libs/types/types.js';
+
+import {
+  type HTTPCode,
+  HTTPError,
+  HTTPHeader,
+  type HTTPLoadParameters,
+} from './libs/types/types.js';
+
+class HTTPService {
```

Comment:
still not fixed it should be almost the same as on frontend, where we have two packages http and api with baseHttpApi

Reply:
working on this